docs/nlu/components.rst

```diff
@@ -526,55 +536,89 @@ CountVectorsFeaturizer
             Unseen words will be substituted with ``OOV_token`` **only** if this token is present in the training
             data or ``OOV_words`` list is provided.
 
-    Sharing Vocabulary between user message and labels:
-
-        .. note:: Enabled only if ``use_shared_vocab`` is ``True``
-
-        Build a common vocabulary set between tokens in labels and user message.
+    If you want to share the vocabulary between user messages and intents, you need to set the option
+    ``use_shared_vocab`` to ``True``. In that case a common vocabulary set between tokens in intents and user messages
+    is build.
 
     .. code-block:: yaml
 
         pipeline:
         - name: "CountVectorsFeaturizer"
-          # whether to use a shared vocab
-          "use_shared_vocab": False,
-          # whether to use word or character n-grams
-          # 'char_wb' creates character n-grams only inside word boundaries
-          # n-grams at the edges of words are padded with space.
-          analyzer: 'word'  # use 'char' or 'char_wb' for character
-          # the parameters are taken from
-          # sklearn's CountVectorizer
-          # regular expression for tokens
-          token_pattern: r'(?u)\b\w\w+\b'
-          # remove accents during the preprocessing step
-          strip_accents: None  # {'ascii', 'unicode', None}
-          # list of stop words
-          stop_words: None  # string {'english'}, list, or None (default)
-          # min document frequency of a word to add to vocabulary
-          # float - the parameter represents a proportion of documents
-          # integer - absolute counts
-          min_df: 1  # float in range [0.0, 1.0] or int
-          # max document frequency of a word to add to vocabulary
-          # float - the parameter represents a proportion of documents
-          # integer - absolute counts
-          max_df: 1.0  # float in range [0.0, 1.0] or int
-          # set ngram range
-          min_ngram: 1  # int
-          max_ngram: 1  # int
-          # limit vocabulary size
-          max_features: None  # int or None
-          # if convert all characters to lowercase
-          lowercase: true  # bool
-          # handling Out-Of-Vacabulary (OOV) words
-          # will be converted to lowercase if lowercase is true
-          OOV_token: None  # string or None
-          OOV_words: []  # list of strings
+          # Analyzer to use, either 'word', 'char', or 'char_wb'
+          "analyzer": "word"
+          # Set the lower and upper boundaries for the n-grams
+          "min_ngram": 1
+          "max_ngram": 1
+          # Set the out-of-vocabulary token
+          "OOV_token": "_oov_"
+          # Whether to use a shared vocab
+          "use_shared_vocab": False
+
+    .. container:: toggle
+
+        .. container:: header
+
+            The above configuration parameters are the ones you most likely gonna change.
+            However, additional parameters exists that can be adapted.
```

Comment:
```suggestion
            The above configuration parameters are the ones you should configure to fit your model to your data.
            However, additional parameters exist that can be adapted.
```
too much? let's change this everywhere when we decide what to say

Reply:
This is my last open comment -- regardless of how we want to phrase this, in the 3 places we say this we should change
`gonna` -> `going to`
`exists` -> `exist`

I thought it was weird to say that someone is "most likely going to change" something, cause we have no idea what the'll do, but we can also leave it like that as long as the grammar is fixed 🤷‍♀️